tests/integration/buildcmd/build_integ_base.py

```diff
@@ -100,7 +100,7 @@ def _verify_resource_property(self, template_path, logical_id, property, expecte
             self.assertEquals(expected_value, template_dict["Resources"][logical_id]["Properties"][property])
 
     def _verify_invoke_built_function(self, template_path, function_logical_id, overrides, expected_result):
-        LOG.info("Invoking built function '{}'", function_logical_id)
+        LOG.info("Invoking built function '{}'".format(function_logical_id))
```

Comment:
thanks for the fix ;) this is probably me. I can't remember how to do interpolation in python :P